torch/distributed/distributed_c10d.py

```diff
@@ -393,7 +393,12 @@ def init_process_group(backend,
             the process group. Default value equals 30 minutes.
             This is applicable for the ``gloo`` backend. For ``nccl``, this is
             applicable only if the environment variable ``NCCL_BLOCKING_WAIT``
-            is set to 1.
+            or ``NCCL_ASYNC_ERROR_HANDLING`` is set to 1. When
+            ``NCCL_BLOCKING_WAIT`` is set, this is the duration for which the
+            process will block and wait for collectives to complete before
+            throwing an exception. When ``NCCL_ASYNC_ERROR_HANDLING`` is set,
```

Comment:
What if both `NCCL_ASYNC_ERROR_HANDLING` and `NCCL_ASYNC_ERROR_HANDLING` are set to 1? Or is this case not allowed?

Reply:
This should be avoided, but NCCL_BLOCKING_WAIT will catch the error first and throw an exception.

Reply:
* Maybe we can add a comment ``NCCL_BLOCKING_WAIT`` is set to 1. or ``NCCL_ASYNC_ERROR_HANDLING`` is set to 1 **(these two configurations are exclusive)**

* which the process **(host thread)** will block and